### DISCUSSION TITLE
insurance/2024-06-25-%E4%B8%AA%E4%BA%BA%E6%89%80%E5%BE%97%E7%A8%8E

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#862] [Announcements]
# insurance/2024-06-25-%E4%B8%AA%E4%BA%BA%E6%89%80%E5%BE%97%E7%A8%8E

我国的税收一共18种，本文只介绍个人所得税。 个人所得税：唯一以自然人为维度缴纳的税种，中国人在全世界范围内的收入都要纳税。 个人所得税 《个人所得税法》规定，下列各项个人所得，应当缴纳个人所得税： （一）工资、薪金所得； （二）劳务报酬所得； （三）稿酬所得； （四）特许权使用费所得； （五）经营所得； （六）利息、股息、红利所得； （七）财产租赁所...

https://moyusisi.github.io/insurance/2024-06-25-%E4%B8%AA%E4%BA%BA%E6%89%80%E5%BE%97%E7%A8%8E.html

**Comment by rgkjhshi** [comment]
test
